keps/sig-windows/1981-windows-privileged-container-support/kep.yaml

```diff
@@ -1,16 +1,20 @@
 title: Windows Privileged Container Support
 kep-number: 1981
 authors:
-  - "@ambguo"
+  - "@ambguo" 
+  - "@marosset"
 owning-sig: sig-windows
 participating-sigs:
   - sig-node 
```

Comment:
sig-auth for security-context implications.
sig-storage for volume mount implications.